Developers/Newcomers_Guide.rst

```diff
@@ -184,3 +184,8 @@ they are really rewarding!
 
     If you need help picking up an issue, you can always ask us and we'll help
     you!
+
+.. note::
+
+    You can also set a profile picture on github so that you can be distinguish
```

Comment:
"You could also add a profile picture on your Github account, so that you can be distinguished out from the crowd!"